src/functions/module_lookup.ts

```diff
@@ -32,6 +38,67 @@ function assertAvailable(variable, searchKey) {
   }
 }
 
+// -----------------------------------------------------------------------------
+// ADDRESS
+// -----------------------------------------------------------------------------
+
+export const ADDRESS: AddFunctionDescription = {
+  description: _lt("Returns a cell reference as a string. "),
+  args: [
+    arg("row (number)", _lt("The row number of the cell reference. ")),
+    arg(
+      "column (number)",
+      _lt("The column number (not name) of the cell reference. A is column number 1. ")
+    ),
+    arg(
+      `absolute_relative_mode (number, default=${ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE})`,
+      _lt(
+        "An indicator of whether the reference is row/column absolute. 1 is row and column absolute (e.g. $A$1), 2 is row absolute and column relative (e.g. A$1), 3 is row relative and column absolute (e.g. $A1), and 4 is row and column relative (e.g. A1)."
+      )
+    ),
+    arg(
+      "use_a1_notation (boolean, default=TRUE)",
+      _lt(
+        "A boolean indicating whether to use A1 style notation (TRUE) or R1C1 style notation (FALSE)."
+      )
+    ),
+    arg(
+      "sheet (string, optional)",
+      _lt("A string indicating the name of the sheet into which the address points.")
+    ),
+  ],
+  returns: ["STRING"],
+  compute: function (
+    row: PrimitiveArgValue,
+    column: PrimitiveArgValue,
+    absoluteRelativeMode: PrimitiveArgValue = ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE,
+    useA1Notation: PrimitiveArgValue = true,
+    sheet: PrimitiveArgValue | undefined
+  ): string {
+    const rowNumber = strictToInteger(row);
+    const colNumber = strictToInteger(column);
+    assertNumberGreaterThanOrEqualToOne(rowNumber);
+    assertNumberGreaterThanOrEqualToOne(colNumber);
+    const _absoluteRelativeMode = strictToInteger(absoluteRelativeMode);
+    assert(
+      () => Object.values(ABSOLUTE_RELATIVE_MODE).includes(_absoluteRelativeMode),
+      expectNumberRangeError(1, 4, _absoluteRelativeMode)
+    );
+    const _useA1Notation = toBoolean(useA1Notation);
+    let cellReference: string;
+    if (_useA1Notation) {
+      cellReference = toA1Notation(rowNumber, colNumber, _absoluteRelativeMode);
+    } else {
+      cellReference = toR1C1Notation(rowNumber, colNumber, _absoluteRelativeMode);
+    }
```

Comment:
I wonder if the helpers aren't overkill here: I agree it helps to understand here, it's good to decompose, but I'm not sure, these helpers will ever be reused.

If we have the opportunity to concentrate the business logic of each formula, let's do it because there are many formulas.

Perhaps inserting the logic directly into the "if" with a shorter code will avoid the creation of new helpers. 
It would give something like this:

```javascript
if (_useA1Notation) {
  rangePart = { 
    rowFixed: [1,2].includes(absoluteRelativeMode) ? true : false,
    colFixed: [1,3].includes(absoluteRelativeMode) ? true : false,
  }
  cellReference = toXC(colNumber-1, rowNumber-1, rangePart)
} else {
  rowPart = [1,2].includes(absoluteRelativeMode) ? `R${rowNumber}` : `R[${rowNumber}]`;
  colPart = [1,3].includes(absoluteRelativeMode) ? `C${colNumber}` : `C[${colNumber}]`;
  cellReference = rowPart + colPart;
}
```

wait to see what others think before considering this change as relevant

Reply:
I agree :+1: 
Even more given those modes that are unlikely to be used elsewhere